torch_optimizer/yogi.py

```diff
@@ -82,9 +90,21 @@ def step(self, closure: OptLossClosure = None) -> OptFloat:
                 if len(state) == 0:
                     state['step'] = 0
                     # Exponential moving average of gradient values
-                    state['exp_avg'] = torch.zeros_like(p)
+                    state['exp_avg'] = nn.init.constant_(
+                            torch.empty_like(
+                                p.data,
+                                memory_format=torch.preserve_format
+                                ),
+                            group["initial_accumulator"]
```

Comment:
Linter complains on double quotes (it enforces consistent quotes in codebase), can you please change it to single ones?

```
flake8 torch_optimizer tests examples setup.py
362torch_optimizer/yogi.py:98:35: Q000 Remove bad quotes
363torch_optimizer/yogi.py:106:35: Q000 Remove bad quotes
364Makefile:5: recipe for target 'flake' failed
365make: *** [flake] Error 1
```
you can run linter locally by doing `make flake`